From 42b2e04bc7a97835386ab38a3e3b7e3dd44dab8a Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@freescale.com>
Date: Fri, 9 Oct 2015 16:35:26 +0800
Subject: [PATCH 0632/5242] MLK-11571 [MX7D-12LPDDR3] LCD: Screen still shows
 with weak backlight after system entered suspend
 mode. 100%

commit  13b78e29f1b2d6dd4a493e10174f2fe32465a5e2 from
https://source.codeaurora.org/external/imx/linux-imx.git

The 'backlight' and 'pwm1' are disabled due to the lpsr mode
not ready in the previous patch in the imx7d-12x12-lpddr3-arm2
dts. So they should be enabled again after lpsr ready.

Signed-off-by: Fancy Fang <chen.fang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
index b8eae33..4d26f0e 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
@@ -19,7 +19,7 @@
 		pwms = <&pwm1 0 5000000>;
 		brightness-levels = <0 4 8 16 32 64 128 255>;
 		default-brightness-level = <6>;
-		status = "disabled";
+		status = "okay";
 	};
 
 	gpio-keys {
@@ -890,7 +890,7 @@
 &pwm1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_pwm1>;
-	status = "disabled";
+	status = "okay";
 };
 
 &sim1 {
-- 
1.7.9.5

